src/main/scala/com/intersys/mdc/challenge/exercises/problems/Problem3.scala

```diff
@@ -25,10 +26,28 @@ case object Problem3 extends Problem {
     * Response: 1<br>1 1<br>1 2 1<br>1 3 3 1
     */
 
+  def getPascalTriangle(size: Long): String = getPascalTriangleAcc(size, Seq[Long](), "")
+
+  @tailrec
+  private def getPascalTriangleAcc(left: Long, prev: Seq[Long], acc: String): String = {
+    val curr: Seq[Long] = prev.size match {
+      case 0 => Seq(1L)
+      case 1 => Seq(1L, 1L)
+      case _ => Seq(1L) ++ prev.sliding(2).toList.map {case Seq(n1, n2) => n1 + n2} ++ Seq(1L)
```

Comment:
Puedes reemplazar `Seq(1L) ++ prev.sliding(2).toList.map {case Seq(n1, n2) => n1 + n2} ++ Seq(1L)` por `Seq(1L)++l.sliding(2).map(_.sum)++Seq(1)`

Reply:
Agree. Thanks for that